README.md

```diff
@@ -30,8 +30,27 @@ for:
 6. Cross Platform: Allows projects to be easily managed across different
    environments.
 
+## Maven and Gradle
+
+Maven and Gradle are two popular build automation tools. 
+
+- Maven is based on developing Java projects. Gradle is based on developing
+domain-specific language (DSL) projects and can build applications of any kind, including Java.
+- Maven uses XML to structure a project. Gradle uses a DSL based on Groovy or Kotlin.  
```

Comment:
Should we explain more on what Groovy and Kotlin are? Or do you plan to elaborate on them a little when you cover Gradle?

Reply:
I just added a brief explanation to this lesson and will provide more explanation in the Gradle lesson.

Reply:
Thanks!